### PR TITLE
Write all csv file generators, add dummydata

### PR DESCRIPTION
**Pull request by paprikatie**

### CODE REVIEW

File: cassandra/generator/photos-generator.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  for (var i = 0; i < 100000; i++) {
+    i === 1000
+      ? console.log('o-o   o-o  o--o o---o ')
+        : i === 2000
+        ? console.log('o     o     |       /')
+          : i === 5000
+          ? console.log('|  -o |  -o O-o   -O-
+          ')
+            : i === 8000
+            ? console.log('o   | o   | |     /
```
Comment (Rdecoster) [inline cassandra/generator/photos-generator.js]: what is going on here with these console logs?  are you using regex?

File: postgres/post_schema.sql [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  url_path text,
+  user_id int REFERENCES users(id),
+);
+
+-- to allow multiple categories per photo:
+  -- join table photos_categories (photo_id, category_id)
+
+-- Import database:
+-- psql toptable -U owner <database/post_schema.sql
+
```
Comment (Rdecoster) [inline postgres/post_schema.sql]: great job here.

File: postgres/generator/users-generator.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      id: counter++,
+      first_name: firsts[randomize(0, firsts.length - 1)];
+      last_name: lasts[randomize(0, lasts.length - 1)];
+      user_avatar: getRandomPhotoUrl();
+    })
+  }
+  writer.end();
+  console.log('Seeded PostgreSQL with 10000 Users!');
+};
+
```
Comment (Rdecoster) [inline postgres/generator/users-generator.js:33]: good job at separations of concerns for each helper function.

**Review by Rdecoster — DISMISSED** [review]
Great Job. Really liked how you kept all of your helper functions in their own js file for seeding functions.

File: cassandra/generator/photos-generator.js (hunk 1 above)
Comment (paprikatie) [reply cassandra/generator/photos-generator.js]: ASCII generator for fun console.log output to show progress in csv generation!

**Pull request merged**